applications/dashboard/src/scripts/app/user-content/embeds/soundcloud.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license https://opensource.org/licenses/GPL-2.0 GPL-2.0
+ */
+
+import { registerEmbed, IEmbedData, FOCUS_CLASS } from "@dashboard/embeds";
+
+// Setup soundcloud embeds.
+registerEmbed("soundcloud", soundCloudRenderer);
+
+/**
+ * Renders soundcloud embeds.
+ */
+export async function soundCloudRenderer(element: HTMLElement, data: IEmbedData) {
+    const height = data.height ? data.height : "";
+
+    const iframe = document.createElement("iframe");
+    iframe.setAttribute("id", "sc-widget");
+    iframe.setAttribute("width", "100%");
+    iframe.setAttribute("height", height.toString());
```

Comment:
Is it necessary to convert this to a string? If so, wouldn't a better place to do so be in the assignment? It looks like you could potentially have an empty string as the default. I don't think you'd want to invoke `toString` on an empty string.